uber/config.py

```diff
@@ -106,18 +106,18 @@ def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
-            if dt or c.HARDCORE_OPTIMIZATIONS_ENABLED:
+            for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
+                if (dt or sa.localized_now()) >= day:
+                    price = bumped_price
+                    # If we set a price during the event, it should be used regardless of badge sales
+                    if c.EPOCH >= day >= c.ESCHATON:
```

Comment:
I've only just had a chance to review this, but unless I'm missing something I think this is wrong in two different ways:

1) It should be ``<=`` rather than ``>=``, unless I'm mistaken.  You're trying to say "if day is between the epoch and eschaton", but the way you have it is saying "if day is before the epoch and after the eschaton" which can literally never happen.

2) Even once you fix that, it's still always going to return the first day after the epoch, i.e. it will always return the Thursday price, because the ``return`` statement will kick in before it has a chance to get to the current day.

What we should probably do then is delete this if-statement from this loop and then after this ``for`` loop, have something like this:

```
# If we set a price during the event, it should be used regardless of badge sales
if c.EPOCH <= sa.localized_now() <= c.ESCHATON:
    return price
```